grakn-test/src/test/java/ai/grakn/test/graph/ConcurrencyTest.java

```diff
@@ -1,158 +0,0 @@
-/*
- * Grakn - A Distributed Semantic Database
- * Copyright (C) 2016  Grakn Labs Limited
- *
- * Grakn is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Grakn is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
- */
-
-package ai.grakn.test.graph;
-
-import ai.grakn.GraknGraph;
-import ai.grakn.concept.Entity;
-import ai.grakn.concept.RelationType;
-import ai.grakn.test.AbstractRollbackGraphTest;
-import ai.grakn.concept.EntityType;
-import ai.grakn.concept.RoleType;
-import ai.grakn.exception.GraknValidationException;
-import org.junit.Ignore;
-import org.junit.Test;
-
-import java.util.HashSet;
-import java.util.Set;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assume.assumeFalse;
-
-/**
- *
- */
-public class ConcurrencyTest extends AbstractRollbackGraphTest {
```

Comment:
No this test was rendered invalid by internalising the ids. So I just deleted it.